nio_cli/commands/buildrelease.py

```diff
@@ -39,6 +40,20 @@ def run(self):
 
         self._write_repo_release()
 
+    def parse_url_from_git_remote_command(self, git_remote):
```

Comment:
This is a static method

Reply:
That's true but it's never used as one. What's the right thing to do here then? Just because it could be used as a static method, does that mean you're supposed to actually make it one?

Reply:
I guess it doesn't matter if it's not being called from an instance?